### ISSUE TITLE
Lower limit function based on common misconception

### ISSUE THREAD

**Issue by Azirine** [mhaeuser/Battery-Toolkit#48]
Contrary to popular belief, micro-cycles are beneficial for the performance of the Li-ion cells. The paper below found that micro-cycles (<2% DOD) increase lifespan in the range of 31% to 50%.

https://www.sciencedirect.com/science/article/pii/S2352152X2201338X

The lower limit function can inadvertently speed up the degration of numerous MacBook batteries, despite users' best intentions.

**Comment by mhaeuser** [comment]
I don’t think you can _easily_ conclude this at all. This is about comparing deep-only vs charging behaviours that also include micro-cycles during regular discharge. If you hit the lower limit (default is -5%), either you briefly discharged and then returned to idle, or it is “creep”-discharging (might even be imprecision with the reading). Meanwhile, the concept of hysteresis is present in Apple’s implementation and I copied their limits. Unless you can extract information that clearly applies to this different use-case from the data (I don’t have the time to check it all in detail), I trust Apple’s internal decisions regarding their own batteries over experiments on only a few cells with a clearly different workload.
